### ISSUE TITLE
Support Vite 5

### ISSUE THREAD

**Issue by bdbvb** [davidmyersdev/vite-plugin-node-polyfills#55]
Add support for Vite 5.

Current configuration:

```
"peerDependencies": {
    "vite": "^2.0.0 || ^3.0.0 || ^4.0.0"
},
```

In the meantime, for a temporary workaround (in your own package.json):

```
"overrides": {
    "vite": "5.0.0"
}
```

**Comment by acuD1** [comment]
Hi, i'm new to npm ecosystem but shouldn't be accurate to write : 

```json
  "overrides": {
    "vite-plugin-node-polyfills": {
      "vite": "^5.0.1"
    }
  }
 ```
 
 @bdbvb

**Comment by daiky00** [comment]
@maxnowack Wehn do you think you can this merge I really need this package for Vite 5?

**Comment by maxnowack** [comment]
Hey @daiky00, I mentioned this issue from another repository. The PR on [SignalDB](https://github.com/maxnowack/signaldb/) doesn't fix this issue, but is also just waiting for a fix, since it's blocking the PR from which I mentioned it. It's just a reference. 

Fixing this issue should be done by the maintainer of `vite-plugin-node-polyfills` @davidmyersdev

**Comment by TomPlum** [comment]
I was gonna raise a PR but I'm getting issues with PNPM when setting up locally. Unsure if its an issue with my local environment. Bumping vite causes a couple of other deps to need to be bumped too. One of them also has the same peer-dependency issue downstream. Build and tests are failing after that though so probably needs someone more experienced with this stuff to have a look.

The override works for now if I bump other vite plugins.

**Comment by maxnowack** [comment]
@TomPlum it seems that is was only on your environment. Installation, building and also the tests are working/passing on my side. I just opened #57 that adds vite v5 to the peer dependencies

**Comment by TomPlum** [comment]
Thanks @maxnowack - I'll have a look at sorting my local 👍🏻

**Comment by daiky00** [comment]
Hey @davidmyersdev, can you review this and get this merge? I really need the polyfills since I am migrating from Webpack to Vite 5.

**Comment by davidmyersdev** [comment]
This was released in [`v0.17.0`](https://github.com/davidmyersdev/vite-plugin-node-polyfills/releases/tag/v0.17.0).
